emsdk.py

```diff
@@ -2587,6 +2592,15 @@ def expand_sdk_name(name):
   elif name in ('latest-upstream', 'latest-clang-upstream', 'latest-releases-upstream'):
     return str(find_latest_releases_sdk('upstream'))
   elif name in ('tot', 'sdk-tot', 'tot-upstream'):
+    if activating:
+      # When we are activating a tot release, assume that the currently
+      # installed SDK, if any, is the tot release we want to activate.
+      # Without this `install tot && activate tot` will race with the builders
+      # that are producing new builds.
+      installed = get_installed_sdk_version()
+      if installed:
+        debug_print('activating currently installed SDK; not updating tot version')
```

Comment:
would this warning not look worrying to people? this goes in the other way of my last comment I guess. but we'd be showing this to most of our users if they enable debug printing. I guess that is rare?

Reply:
Yes this is debug message, not a warning, so most folks will never see it.